css/_custom-mixins.scss

```diff
@@ -0,0 +1,8 @@
+@mixin font-rule($font-family, $font-weight, $font-size, $color, $line-height: 1.3, $transform: none) {
+  font-family: $font-family;
+  font-weight: $font-weight;
+  font-size: $font-size;
+  line-height: $line-height;
+  color: $color;
+  text-transform: $transform;
+}
```

Comment:
Do you really need this mixin?
How we can use this mixin if we need all of this properties except of text-transform, for example?
I mean all of the variables for this mixin are required, and it is not good.